infrastructure/docker-compose.production-deploy.yml

```diff
@@ -1,117 +1,160 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
-#
-# OpenCRVS is also distributed under the terms of the Civil Registration
-# & Healthcare Disclaimer located at http://opencrvs.org/license.
-#
-# Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
 version: '3.3'
 
 services:
-  logstash:
+  gateway:
+    environment:
+      - NODE_ENV=production
+      - LANGUAGES=en,fr
+      - SENTRY_DSN=${SENTRY_DSN}
     deploy:
       replicas: 2
 
-  apm-server:
+  workflow:
+    environment:
+      - NODE_ENV=production
+      - LANGUAGES=en,fr
+      - SENTRY_DSN=${SENTRY_DSN}
     deploy:
       replicas: 2
-    
-  client:
+
+  search:
+    environment:
+      - NODE_ENV=production
+      - SENTRY_DSN=${SENTRY_DSN}
     deploy:
       replicas: 2
 
-  components:
+  metrics:
+    environment:
+      - QA_ENV=true
+      - NODE_ENV=production
+      - SENTRY_DSN=${SENTRY_DSN}
+      - MONGO_URL=mongodb://metrics:${METRICS_MONGODB_PASSWORD}@mongo1,mongo2/metrics?replicaSet=rs0
```

Comment:
**Breaking change:**
- This would have to be changed back to 3 server configuration by a country pulling the repository